explainer-server/app/views/explainList.scala.html

```diff
@@ -1,28 +1,36 @@
 @(explainers: Seq[shared.Explainer])(implicit request: RequestHeader)
 
+@import org.joda.time.DateTime
+@import org.joda.time.format.DateTimeFormat
+
 @header = {
     <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/explainer.css")">
 }
 @templates.main("Explainers", header){
     <div id="dashboard">
         <div id="dashboard__title"><h2>Explainers</h2></div>
         <div id="dashboard__header">
-            <button onclick="ExplainEditorJS().CreateButtonClick()">Create New</button>
+            <button onclick="ExplainEditorJS().CreateNewExplainer()">Create New</button>
         </div>
         <div id="dashboard__body">
             <div id="dashboard__body__header">
-                <div class="title">Title</div>
-                <div class="last_modified">Last modified</div>
-                <div class="created_by">Created by</div>
-                <div class="status">Status</div>
+                <div class="title"><b>Title</b></div>

```

Comment:
you could give all these fields a 'heading' class then set the bold style (and anything else) on that